doc/api/cli.md

```diff
@@ -553,6 +553,10 @@ if they had been specified on the command line before the actual command line
 (so they can be overridden). Node.js will exit with an error if an option
 that is not allowed in the environment is used, such as `-p` or a script file.
 
+In case an option happens to contain a space or a backslash (for example within
+the path passed to `--require`), they must be escaped using an additional
+backslash.
```

Comment:
How about just this instead?:

```markdown
Use a backslash (`\`) to escape all spaces and backslashes within options.
```

Reply:
Original text could be clearer, but I'm blanking on how. You don't want to do all, just the ones in option values. For example, `--throw-deprecation --trace-deprecation`, if it had all spaces escaped, would be `--throw-deprecation\ --trace-deprecation`, which join the strings into an unknown option name.

Reply:
I'm concerned about any logic that would apply only on option values.

- It makes the parsing more complex
- Escaping option names break them, but that's very easy to catch
- It's unexpected - why would it work in a place but not in another?
- Assuming you later want to support a `--key=value` syntax, it would require extra parsing

In general, it sounds more future proof. However unlikely it is that an option would have a space, there are no reason to add a specific restriction which would have no value by itself.

Reply:
I don't think Rich is suggesting you modify the code. No option name will ever have a space in it, so the code can continue (as it does now) to map `--bad\ option` to an option named `"bad option"`... which will then fail later because no option of that name exists. 

For what its worth, I am OK with your original text.

Reply:
Oh sure, I thought you meant that (because of "You don't want to do all, just the ones in option values") 😃

Reply:
@arcanis I think this is one outstanding comment that you should address (just update the text explaining how to escape) that was holding it up landing.

Reply:
Updated the wording